lua/apisix/consumer.lua

```diff
@@ -35,6 +36,10 @@ local function plugin_consumer()
     end
 
     for _, consumer in ipairs(consumers.values) do
+        if type(consumer) ~= "table" then
```

Comment:
```
if type(consumer) == "table" then
    for ...
end
```
is better.

Reply:
The current code style is to reduce indentation. I prefer the current coding style.